tailwind.config.js

```diff
@@ -37,8 +37,8 @@ module.exports = {
 			mini: { min: "260px", max: "300px" },
 			// => @media (min-width: 260px and max-width: 300px) { ... }
 
-			M: { min: "500px", max: "720px" },
-			// => @media (min-width: 540px and max-width: 720px) { ... }
+			M: { min: "600px", max: "720px" },
+			// => @media (min-width: 500px and max-width: 720px) { ... }
```

Comment:
I would rework these, there is some size overlapping here (mini and sm both act in the 260 to 300 range)
you should keep it simple, I recommend doing no more than 3 sizes:
- sm (0 to 767) - phones
- md (768 to 1023) - tablets
- lg (1024 and up) - desktops

in case you have specific designs for full hd screens, which I don't think you do for this project, you can add an extra one for those
- lg (1024 to 1279) - desktops
- xl (1280 and up) - full hd displays and up

those are the most commonly used screen sizes